qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: 1.1. Sign in to App with Google account
+
+Scenario: Sign in with Google account granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	When user hits "Sign in with Google" button
+	And "Google account authentication" is granted by Google
+	Then user is signed in to the app
+	And user is taken to "Dataset" page of the app
+
+Scenario: Sign in with Google account is not granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	When user hits "Sign in with Google" button
+	And "Google account authentication" is not granted by Google
+	Then user sign in to the app is denied
+	And user is taken to "Sign in with Google" page
```

Comment:
probably there could be some examples/data tables to provide some exact data for signing in process 
but i'm not sure whether it's necessary

Reply:
1) for this scenario the same comments as for the previous one
2) We should use specific data for testing: emails, passwords in order to check different cases